modules/nf-core/vt/decomposeblocksub/main.nf

```diff
@@ -0,0 +1,59 @@
+process VT_DECOMPOSEBLOCKSUB {
+    tag "$meta.id"
+    label 'process_low'
+
+    // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vt:2015.11.10--h5ef6573_4':
+        'biocontainers/vt:2015.11.10--h5ef6573_4' }"
+
+    input:
+    tuple val(meta), path(vcf), path(index), path(intervals)
+
+    output:
+    tuple val(meta), path("*.vcf.gz")   , emit: vcf
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def args2 = task.ext.args2 ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    if ("$vcf" == "${prefix}.vcf" || "$vcf" == "${prefix}.vcf.gz") {
```

Comment:
The vcf check is not necessary as you force it to compressed format.
```suggestion
    if ("$vcf" == "${prefix}.vcf.gz") {
```

Reply:
input file can also be vcf (both is accepted by the tool)